docs/user/faq.rst

```diff
@@ -342,3 +342,80 @@ you can manually convert your Jupyter Book project to Sphinx with the following
             pre_build:
             # Generate the Sphinx configuration for this Jupyter Book so it builds.
             - "jupyter-book config sphinx docs/"
+
+
+Common build errors and solutions
+---------------------------------
+
+This section is very specific and may change over time.
+Please help us keep it updated and contribute your own build errors and resolutions.
+
+.. This section can improve with sphinx-design and dropdown cards
+
+Git errors
+~~~~~~~~~~
+
+In the examples below, we use ``github.com``, however error messages are similar for GitLab, Bitbucket etc.
+
+
+terminal prompts disabled
+"""""""""""""""""""""""""
+
+.. code-block:: text
+
+   fatal: could not read Username for 'https://github.com': terminal prompts disabled
+
+**Resolution:** This error can be quite misleading. It usually occurs when a repository could not be found because of a typo in the reposistory name or because the repository has been deleted. Verify your repository URL in :guilabel:`Admin > Advanced Settings`.
+
+This error also occurs if you have changed a ``public`` repository to ``private`` and you are using ``https://`` in your git repository URL.
+
+.. note::
+
+   To use private repositories, you need a plan on `Read the Docs for Business <https://readthedocs.com>`__.
+
+
+error: pathspec
+"""""""""""""""
+
+.. code-block:: text
+
+   error: pathspec 'main' did not match any file(s) known to git
+
+**Resolution:** A specified branch does not exist in the git repository.
```

Comment:
Note, there are a few instances of this this, I'd do a find/replace to catch the rest.

```suggestion
**Resolution:** A specified branch does not exist in the Git repository.
```